README.md

```diff
@@ -14,58 +14,73 @@
 [ci-badge]: https://github.com/ngrok/ngrok-nodejs/actions/workflows/ci.yml/badge.svg
 [ci-url]: https://github.com/ngrok/ngrok-nodejs/actions/workflows/ci.yml
 
-**Note: This is beta-quality software. Interfaces may change without warning.**
+`ngrok-nodejs` is the official Node.js SDK for ngrok that requires no binaries. Quickly enable secure production-ready connectivity to your applications and services directly from your code.
```

Comment:
Maybe `separate executable binary`

Reply:
Thought about this, but ultimately decided against it. It felt like it implied there was still a binary bundled.